repro.in

```diff
@@ -374,8 +374,8 @@ __END__
     build_package "build" $builddir
 
     msg "Comparing hashes..."
-    sha512sum -b "$pkg" | read refference_build_checksum _
-    sha512sum -b ./build/"$(basename $pkgfile)" | read other_build_checksum _
+    read -r refference_build_checksum _ < <(sha512sum -b "$pkg")
+    read -r other_build_checksum _ < <(sha512sum -b ./build/"$(basename $pkgfile)")
     if [ "$refference_build_checksum" = "$other_build_checksum" ]; then
```

Comment:
I think the proper bash way is probably:
```sh
msg "Comparing files..."
if diff -q -- "$pkg" ./build/"$(basename $pkgfile)" > /dev/null ; then
  msg "Package is reproducible!"
else
[...]
```

Reply:
Hum. Gotta try out that and see what i like better. Thanks!

Reply:
Also:
```sh
./build/"$(basename $pkgfile)"
```
is buggy and should be
```
./build/"$(basename "$pkgfile")"
```
as it currently doesn't handle spaces in $pkgfile correctly.

Reply:
I have no preference tbh, @Foxboron you might want to move this code to a separate function so it's re-usable for build and check.